clusterloader2/testing/density/config.yaml

```diff
@@ -48,6 +48,10 @@ steps:
     Method: APIResponsiveness
     Params:
       action: reset
+  - Identifier: APIResponsivenessPrometheus
+    Method: APIResponsivenessPrometheus
+    Params:
+      action: start
```

Comment:
We're getting to the point when it makes sense to introduce the "slo bundle" metric. Soon we will have network-programming latency SLI in this categoy, and more is comming - I don't want to grows those configs that much.
Probably a separate PR though.

Reply:
I will create separate PR.